ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -336,6 +338,20 @@ func (r *RayClusterReconciler) reconcilePods(instance *rayiov1alpha1.RayCluster)
 	if err := r.List(context.TODO(), &headPods, client.InNamespace(instance.Namespace), filterLabels); err != nil {
 		return err
 	}
+	if EnableBatchScheduler {
+		var minMember int32
+		var totalResource corev1.ResourceList
+		if instance.Spec.EnableInTreeAutoscaling == nil || !*instance.Spec.EnableInTreeAutoscaling {
```

Comment:
Should we use minMember to indicate `min` of gang or use a separate annotation?

Reply:
For Volcano, it considers `minMember` to be the min gang size, which we set to be the total replica count (if not using autoscaling) or the total minReplica count (when using autoscaling).

Did you want to make this more configurable so that users can specify their own min gang size?